.travis.yml

```diff
@@ -5,7 +5,6 @@ node_js:
   - '8'
   - '6'
   - '4'
-before_install: if [[ `npm -v` < 3* ]]; then npm i -g npm@^3; fi
 after_success:
   - bash <(curl -s https://codecov.io/bash)
   - npm install -g semantic-release
```

Comment:
I'll re-configure our `semantic-release` setup in separate PR. they did a lot of breaking changes so we need to modify our config

Reply:
@sudo-suhas oh yeah, one more thing.. what about [greenkeeper-lockfile](https://github.com/greenkeeperio/greenkeeper-lockfile)?

Reply:
Yes, good point. 

This should do the job:
```yaml
# ...
before_install:
  - yarn global add greenkeeper-lockfile@1

before_script: greenkeeper-lockfile-update

# Only the node version 8 job will upload the lockfile
after_script:
  - greenkeeper-lockfile-upload
```

Reply:
we need to run `yarn install` with `--no-ignore-optional`, right?
or our `yarn.lock` wouldn't be updated as expected
![screen shot 16](https://user-images.githubusercontent.com/22868432/34434092-4f4a00f2-ecb6-11e7-9d9d-e40e9a5bed4a.png)